tests/examples/hello/src/iota.cc

```diff
@@ -0,0 +1,28 @@
+#include "hello_world.h"
+#include "legate_library.h"
+
```

Comment:
I suggest adding a logger (that all tasks in the hello example use), which log information that would help the reader understand what's going on:

```
log.info() << "Task iota handling points " << output_shape.lo << " to " << output_shape.hi;
```